vcs/internal/path.go

```diff
@@ -0,0 +1,13 @@
+package internal
+
+import "strings"
+
+// Rel strips the leading "/" prefix from the path string, effectively turning
+// an absolute path into one relative to the root directory. A path that is just
+// "/" is treated specially, returning just ".".
+func Rel(path string) string {
+	if path == "/" {
+		return "."
+	}
```

Comment:
I will retract this suggestion. As discussed, the reason why things continue to work without the special case here is because other code happens to call `filepath.Clean` which replaces empty string with ".".

However, empty string is not a valid path for `os.Open`, for example, while "." is valid. So we should keep this special case for callers that do not use `filepath.Clean`